packages/flutter/lib/src/material/material_state_mixin.dart

```diff
@@ -0,0 +1,168 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+
+import 'material_state.dart';
+
+/// Mixin for [State] classes that require knowledge of changing [MaterialState]
+/// values for their child widgets.
+///
+/// This mixin does nothing by mere application it to a [State] class, but is
```

Comment:
I think there is a typo here.

```suggestion
/// This mixin does nothing by mere application of it to a [State] class, but is
```